scripts/docker/zeppelin-quick-start/docker-compose-zeppelin-only.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+version: "3.8"
+
+services:
+  zeppelin-server:
+    hostname: zeppelin
+    container_name: zeppelin
+    image: apache/zeppelin:0.11.2
```

Comment:
>  ! zeppelin-server The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

I tested it in an Apple Silicon machine, seems we don't provide arm64 image, this is a thing we can improve later

Reply:
I will complete this task later.